doc/spec/manifest.md

```diff
@@ -0,0 +1,214 @@
+# Distribution Content Manifests
+
+A Content Manifest is a simple JSON file which contains general fields that
+are typical to any package management system. The goal is for these manifests
+to describe an application and its dependencies in a content-addressable and
+verifiable way.
+
+## Field Descriptions
```

Comment:
okay, I've updated all of the field descriptions with an `(optional)` marker if it may be omitted.